### PR TITLE
Library: Author search

### PR DESCRIPTION
**Pull request by synox**
Simple tool to copy author name and id

Test URLs:
- After: https://author-search-library--24life--hlxsites.hlx.page/tools/sidekick/library.html?plugin=authors

### CODE REVIEW

File: tools/sidekick/config.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "project": "24life",
```
Comment (iamitshri) [inline tools/sidekick/config.json:2]: @synox  do we want to adjust the github project settings so PR is hinted to get the latest changes from main?

File: tools/sidekick/plugins/authors/authors.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+:root {
+  --swc-menu-width: 100%;
+}
+
+.container {
```
Comment (iamitshri) [inline tools/sidekick/plugins/authors/authors.css:5]: does need to be scoped to author? like author-container?

File: tools/sidekick/plugins/authors/authors.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  container.append(spContainer);
+
+  spContainer.querySelectorAll('sp-menu-item')
+    .forEach((item) => {
+      item.addEventListener('click', handleCopyButtonClick);
+    });
+}
+
+async function getAuthorData() {
+  const resp = await fetch('/authors.json?limit=5000');
```
Comment (iamitshri) [inline tools/sidekick/plugins/authors/authors.js:40]: was wondering what kind of auth this endpoint uses?

**Review by iamitshri — APPROVED** [review]

File: tools/sidekick/config.json (hunk 1 above)
Comment (synox) [reply tools/sidekick/config.json:2]: The config is described here: https://www.hlx.live/docs/sidekick#customizing-the-sidekick

Comment (iamitshri) [reply tools/sidekick/config.json:2]: Sorry, I meant this file config.json is already present in the main branch.  If the branch is up-to-date with main we wouldn't have seen this file is what I meant :)

Comment (synox) [reply tools/sidekick/config.json]: Ah there was a merge conflict, that's why. Should now look better.

**Pull request merged**